features/dd-sdk-android-rum/api/dd-sdk-android-rum.api

```diff
@@ -6158,3 +6158,474 @@ public final class com/datadog/android/telemetry/model/TelemetryErrorEvent$View$
 	public final fun fromJsonObject (Lcom/google/gson/JsonObject;)Lcom/datadog/android/telemetry/model/TelemetryErrorEvent$View;
 }
 
+public final class com/datadog/android/telemetry/model/TelemetryUsageEvent {
```

Comment:
seems like it shouldn't be a part of this PR?

Reply:
removed